resources/quality/ultimaker3/um3_aa0.4_TPU_Normal_Quality.inst.cfg

```diff
@@ -24,6 +24,7 @@ machine_nozzle_heat_up_speed = 2.5
 material_final_print_temperature = =material_print_temperature - 10
 material_flow = 106
 material_initial_print_temperature = =material_print_temperature - 10
+material_print_temperature = =default_material_print_temperature - 2
```

Comment:
'material_print_temperature = =default_material_print_temperature + 3'
"or" I am not able to figure out how this is calculated